quickstart_lists/builder_x86_64-linux_stable

```diff
@@ -0,0 +1,14 @@
+# Download via:
```

Comment:
WIll this file format work with 0.88?  If not, I propose we ship these files initially without the # comments so that they are immediately useful.